crates/node/src/config.rs

```diff
@@ -223,4 +227,9 @@ impl Configuration {
     fn extract(figment: Figment) -> Result<Self, Error> {
         Ok(figment.extract()?)
     }
+
+    /// Dumps the configuration to a string
+    pub fn dump(&self) -> Result<String, Error> {
+        Ok(serde_yaml::to_string(self)?)
```

Comment:
I guess we are changing this to toml once we move away from yaml, right?

Reply:
Correct.